spec/services/site_setting/update_spec.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+RSpec.describe SiteSetting::Update do
+  describe described_class::Contract, type: :model do
+    it { is_expected.to validate_presence_of :setting_name }
+  end
+
+  describe ".call" do
+    subject(:result) { described_class.call(**params, **options, **dependencies) }
```

Comment:
As this is extracted from https://github.com/discourse/discourse/pull/29129, I kept the `options` let, this will make changes minimal in the big PR :grin: